My first Commit For Code Review

khanh-devos opened the pull request:
- Config linter for html and css
- Add .gitignore
- build a standard README.md
- Add index.html, index.css
- Add test.md but ignore it

khanh-devos committed:
Add index.html file

khanh-devos committed:
Add index.css file

khanh-devos committed:
Add test.md file but Ignore it

tresorsawasawa commented on README.md:
```diff
@@ -1 +1,40 @@
+
+
+
+
```
- [ ] For the project requirements, kindly ensure that your README file is descriptive and professional. And the required template to be used here is the **[Microverse README template](https://github.com/microverseinc/readme-template)**, so ensure that: 
  * All main headers from the [README template](https://github.com/microverseinc/readme-template) are present in a your repo.
  * The only optional section is `FAQ`
  *  you delete the `live demo` section if the you decide not to deploy the application.
  * Make sure the you have added the section `Future Features`, including 1-3 potential features to be added.
  * you remove the section `description`, i.e.` "> Describe the tech stack and include only the relevant sections that apply to your project."` should be removed and filled in with their custom content.
  * you remove template logo. Optionally, the you can replace it with your own logo you they have one.
  * No broken link or image

khanh-devos replied:
Thanks for your comment, I will improve it then commit again.

khanh-devos replied:
Hello @tresorsawasawa 
I modified README file already.
But now I cannot change branch name because I try to keep the same request.

taldr27 replied:
Hi, @khanh-devos, great work doing the requested changes, and don't worry about the branch name in this Pull Request, you can't change it because that'll modify the GitHub history, just take the recommendation for your future projects. 💯

tresorsawasawa reviewed (changes requested):
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @khanh-devos  
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

![almost-there](https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif)

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [x] As required for the project, kindly make sure your **Pull Request title** is more descriptive  in the way tells  what changes are inside. e.g.: **"Milestone: Set up a Hello Microverse project"**

- [x] Also for the project requirements, kindly ensure that  your `branch` has a name of a `feature` or action (e.g. 'created_sidebar' or 'added_comments_section',...). As the current one is not enough descriptive, you can checkout to `new branch`, `use the correct name` and `mention the old PR` in the new PR.

- [x] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

khanh-devos committed:
Modify READE file

- change Tech Stack
- Delete Live demo
- Add more Setup, Intall, Run tests,
- Add Future Features

taldr27 commented on README.md:
```diff
@@ -1 +1,46 @@
+
+License:
+    This project is MIT licensed.
+
```
- [ ] As requested by the previous reviewer, it's recommended to use the [README template](https://github.com/microverseinc/readme-template) for your repository's `README`. You can easily copy the template by following these steps:

### Step 1: Go to [this link](https://github.com/microverseinc/readme-template) and click on the README.md file as in the image:
![image](https://user-images.githubusercontent.com/107952724/234482413-a83503c4-4afe-4506-a588-64722805daed.png)

### Step 2: Click on the copy raw file or if you want you can download it and paste it into your project.
![image](https://user-images.githubusercontent.com/107952724/234482958-89a99a38-71c4-498b-87cc-870192cbf229.png)

### Step 3: Customize the template with your project information, and also don't forget to remove the commented code and the logo image. 👌

Following these steps will help you create a more professional `README` file.

khanh-devos replied:
Hello Diego Garcia, 
thanks alot for your specific review

taldr27 reviewed (changes requested):
## Hi @khanh-devos, 

Great work on making the changes requested by a previous reviewer 👏 👏 
You've done well implementing some of the requested changes, but some are still not addressed yet.

![almost](https://media.giphy.com/media/t8YPf3ACKeouLzEIvr/giphy.gif)

## Highlights 🎉
✅ Linters tests passing.
✅ Correct use of GitHub Flow.

 ## Required Changes ♻️
_Check the comments under the review._

------
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in @taldr27 your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). <img align="right" src="https://media.giphy.com/media/kReKcfrs1YoTmt2AQt/giphy.gif" alt="gif"  width="3%"/>_

khanh-devos committed:
use exact README file Template

AbelG101 commented on README.md:
```diff
@@ -1 +1,185 @@
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
- [x] Ouch this is so close but it seems like the MIT link is broken check the video below:
  ![firefox_nHQ4CD7Fe6](https://user-images.githubusercontent.com/104968236/234548735-ee38443d-0903-4526-bfa6-b6f9fabd387e.gif)
- So this issue is occurring because you don't have a LICENSE file at your root repository, you can add the MIT.md file [found here,](https://github.com/microverseinc/readme-template/blob/master/MIT.md), customize the content and replace the `./LICENSE` in line `183` with `./MIT.md` 
```suggestion
This project is [MIT](./MIT.md) licensed.
```

khanh-devos replied:
Yeah !!!, thank you a lot, Mr.Abel

AbelG101 reviewed (changes requested):
![](https://media.giphy.com/media/7SyGkO0kQWCfnUC47Y/giphy.gif)


# Hi @khanh-devos 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

 ## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

khanh-devos committed:
fix MIT license link

khanh-devos committed:
add more FAQ at README file

franclobo commented on README.md:
```diff
@@ -1 +1,197 @@
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [your_project_name] <a name="about-project"></a>
```
- [x] May you forget to replace `your_project_name` with `Hello world`? Please 🙏🏽 do it for professional documentation.

franclobo commented on README.md:
```diff
@@ -1 +1,197 @@
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```
- [x] To avoid the 404 error, please 🙏🏽 add the LICENSE file to your repo and ensure that the path is the correct 👉🏽 `./LICENSE` for professional documentation. You can use these [instructions](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository).

franclobo reviewed (changes requested):
Hi @khanh-devos, 

# ♻️ Required Changes ♻️

![soClose](https://user-images.githubusercontent.com/58642949/234582356-7e716204-5701-422b-aa23-d235ed182939.gif)

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Github flow ✔️ 
- Good communication with CR ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

khanh-devos committed:
Add a LICENSE file and Link README file with it

taldr27 commented on README.md:
```diff
@@ -1 +1,190 @@
+
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
```
- [x] **[OPTIONAL]** Great work on using the `README` template correctly! I have an optional suggestion to make your `README` even better - consider removing any commented-out code to make the file cleaner and more professional. Keep up the good work! 🎯🎯

khanh-devos replied:
Hello @taldr27,
Thank you for my first Approval at Microverse

taldr27 reviewed (approved):
## Hi @khanh-devos, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉

![congrats](https://media.giphy.com/media/3o6fIUZTTDl0IDjbZS/giphy.gif)

## Highlights 🎉
✅ Professional and customized README. 
✅ Linters passing.
✅ Correct use of GitHub Flow.
✅ Incorporates feedback from previous code reviews.
✅ Great communication with reviewers.

## Optional suggestions 💭

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

------
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). <img align="right" src="https://media.giphy.com/media/kReKcfrs1YoTmt2AQt/giphy.gif" alt="gif"  width="3%"/>_
